### ISSUE TITLE
Remove DSLogon status checker

### ISSUE THREAD

**Issue by joeniquette** [department-of-veterans-affairs/va.gov-team#47709]
The tool made by Boris needs to be removed and eventually replaced with a new one.

Current link: https://dslogon.statuspage.io/
This conversation seems to point out the current gaps: https://dsva.slack.com/archives/C01G476SCEN/p1660969562339379

- One of the points made by Boris is that the [backend_status](https://api.va.gov/v0/backend_statuses) for DSLogon isn't correct, and hasnt been since March. Not sure what this means but we need to look into this as well as part of this effort. I believe what leads him to believe this is the case is that there havent been any DSLogon reports sent to [this](https://dsva.slack.com/archives/C02SBFQ22RL/p1647618495070789) channel as would be expected
- Another point made, ebenefits apparently actually relies on this tool. So we need to first confirm with Matt Self about this reliance before we remove the solution. My preference would be to remove it.
- All devops DSLogon monitors as it ties to pagerduty needs to be verified as still working as expected
- Boris states that Nancy has the statuspage subscription, if this is true we need to follow up with her if/when we make any changes. At the minimum we need to remove the login check on the statuspage because it was tied to Boris DSLogon account but that account was deactivated.

**Comment by joeniquette** [comment]
@joeniquette document details

**Comment by ianhundere** [comment]
https://dslogon.statuspage.io/
https://api.va.gov/v0/backend_statuses

**Comment by ianhundere** [comment]
https://github.com/department-of-veterans-affairs/devops/blob/master/terraform/environments/datadog/dslogon.tf
^ this sends a notification to `#qas-notifications` in slack

**Comment by ianhundere** [comment]
![image (7).png](https://images.zenhubusercontent.com/624b40ac4980d90d75917e4c/2fd89196-c36f-4a0d-b86b-adf8df985c1e)

**Comment by ianhundere** [comment]
dslogon.statuspage.io

**Comment by ianhundere** [comment]
Nancy has the login for statuspage.io:
https://dsva.slack.com/archives/C01G476SCEN/p1661212366241259?thread_ts=1660969562.339379&cid=C01G476SCEN

**Comment by ianhundere** [comment]
i msg'd Nancy about getting access to a statuspage.io login.

**Comment by ianhundere** [comment]
https://grafana.vfs.va.gov/d/000000033/site-authentication

**Comment by ianhundere** [comment]
https://dsva.pagerduty.com/incidents/Q267MTJ03F83GH

**Comment by ianhundere** [comment]
i got access to the status page.

**Comment by ianhundere** [comment]
as far as i can tell, the monitor is comprised of the following:
- dd resources (e.g. synthetic test / monitor)
- status.io page

as for parts no longer in use, i don't believe the following script is in use any longer, i think the dd synth test essentially replaced it.:
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/22284 / https://github.com/department-of-veterans-affairs/monitor-dslogon

**Comment by ianhundere** [comment]
tl;dr
- [Boris’ puppeteer script](https://github.com/department-of-veterans-affairs/monitor-dslogon) that looks at `backend_statuses` was/is running on an AWS account with the id of `743568122158`
- we currently have a [datadog synthetic test](https://github.com/department-of-veterans-affairs/devops/pull/11377/files) that looks at `backend_statuses` to check is dslogon is up (this is incorrect)
- the [statuspage.io](https://dslogon.statuspage.io/) points toward Boris’ pingdom account to track response time of the dslogon page (we could easily replace this statuspage metric with our own from datadog so these people can continue monitoring using this statuspage)![image.png](https://images.zenhubusercontent.com/624b40ac4980d90d75917e4c/e9daf847-9def-443a-b78f-dc04065b2eeb)
![image.png](https://images.zenhubusercontent.com/624b40ac4980d90d75917e4c/bcd1afe7-1e82-47e9-8850-73da711edcd1)

**Comment by ianhundere** [comment]
PR opened:
https://github.com/department-of-veterans-affairs/devops/pull/12164

we'll then wanna create a new synthetic test (or http check, whatever) to replace it as well as hook https://dslogon.statuspage.io into the new monitor as well.

**Comment by ianhundere** [comment]
the status emitting from `backend_status` has since been corrected. no need to do this.

**Comment by ianhundere** [comment]
chatted with @joeniquette about how to go about the work in this ticket. i've added an acceptance criteria above to better chunk / plan our efforts.
